### PR TITLE
Add toString in tarantool proxy entities

### LINKED ISSUE #406: Add toString method to TarantooResultImpl, TarantoolTupleImpl and other classes
toString method should be added for demonstration purposes.
In customer workshops, objects should be automatically converted to string.

### PR DESCRIPTION
**Pull request by ArtDu**
<details><summary>before</summary>
<p>

```
io.tarantool.driver.core.TarantoolTupleResultImpl@6a1530b5
```

</p>
</details> 

<details><summary>after</summary>
<p>

```
TarantoolResultImpl{tuples=[TarantoolTupleImpl{
	spaceMetadata=TarantoolSpaceMetadata{spaceId=0, ownerId=0, spaceName='null', spaceFormatMetadata={profile_id=TarantoolFieldMetadataImpl{fieldName='profile_id', fieldType='unsigned', fieldPosition=0, isNullable=false}, bucket_id=TarantoolFieldMetadataImpl{fieldName='bucket_id', fieldType='unsigned', fieldPosition=1, isNullable=false}, fio=TarantoolFieldMetadataImpl{fieldName='fio', fieldType='string', fieldPosition=2, isNullable=false}, age=TarantoolFieldMetadataImpl{fieldName='age', fieldType='unsigned', fieldPosition=3, isNullable=false}, balance=TarantoolFieldMetadataImpl{fieldName='balance', fieldType='unsigned', fieldPosition=4, isNullable=true}}, spaceFormatMetadataAsList=[TarantoolFieldMetadataImpl{fieldName='profile_id', fieldType='unsigned', fieldPosition=0, isNullable=false}, TarantoolFieldMetadataImpl{fieldName='bucket_id', fieldType='unsigned', fieldPosition=1, isNullable=false}, TarantoolFieldMetadataImpl{fieldName='fio', fieldType='string', fieldPosition=2, isNullable=false}, TarantoolFieldMetadataImpl{fieldName='age', fieldType='unsigned', fieldPosition=3, isNullable=false}, TarantoolFieldMetadataImpl{fieldName='balance', fieldType='unsigned', fieldPosition=4, isNullable=true}]},
	fields=[1000000, 14045, FIO, 50, 100]
}, TarantoolTupleImpl{
	spaceMetadata=TarantoolSpaceMetadata{spaceId=0, ownerId=0, spaceName='null', spaceFormatMetadata={profile_id=TarantoolFieldMetadataImpl{fieldName='profile_id', fieldType='unsigned', fieldPosition=0, isNullable=false}, bucket_id=TarantoolFieldMetadataImpl{fieldName='bucket_id', fieldType='unsigned', fieldPosition=1, isNullable=false}, fio=TarantoolFieldMetadataImpl{fieldName='fio', fieldType='string', fieldPosition=2, isNullable=false}, age=TarantoolFieldMetadataImpl{fieldName='age', fieldType='unsigned', fieldPosition=3, isNullable=false}, balance=TarantoolFieldMetadataImpl{fieldName='balance', fieldType='unsigned', fieldPosition=4, isNullable=true}}, spaceFormatMetadataAsList=[TarantoolFieldMetadataImpl{fieldName='profile_id', fieldType='unsigned', fieldPosition=0, isNullable=false}, TarantoolFieldMetadataImpl{fieldName='bucket_id', fieldType='unsigned', fieldPosition=1, isNullable=false}, TarantoolFieldMetadataImpl{fieldName='fio', fieldType='string', fieldPosition=2, isNullable=false}, TarantoolFieldMetadataImpl{fieldName='age', fieldType='unsigned', fieldPosition=3, isNullable=false}, TarantoolFieldMetadataImpl{fieldName='balance', fieldType='unsigned', fieldPosition=4, isNullable=true}]},
	fields=[1000001, 14000, FIO, 51, 101]
}, TarantoolTupleImpl{
	spaceMetadata=TarantoolSpaceMetadata{spaceId=0, ownerId=0, spaceName='null', spaceFormatMetadata={profile_id=TarantoolFieldMetadataImpl{fieldName='profile_id', fieldType='unsigned', fieldPosition=0, isNullable=false}, bucket_id=TarantoolFieldMetadataImpl{fieldName='bucket_id', fieldType='unsigned', fieldPosition=1, isNullable=false}, fio=TarantoolFieldMetadataImpl{fieldName='fio', fieldType='string', fieldPosition=2, isNullable=false}, age=TarantoolFieldMetadataImpl{fieldName='age', fieldType='unsigned', fieldPosition=3, isNullable=false}, balance=TarantoolFieldMetadataImpl{fieldName='balance', fieldType='unsigned', fieldPosition=4, isNullable=true}}, spaceFormatMetadataAsList=[TarantoolFieldMetadataImpl{fieldName='profile_id', fieldType='unsigned', fieldPosition=0, isNullable=false}, TarantoolFieldMetadataImpl{fieldName='bucket_id', fieldType='unsigned', fieldPosition=1, isNullable=false}, TarantoolFieldMetadataImpl{fieldName='fio', fieldType='string', fieldPosition=2, isNullable=false}, TarantoolFieldMetadataImpl{fieldName='age', fieldType='unsigned', fieldPosition=3, isNullable=false}, TarantoolFieldMetadataImpl{fieldName='balance', fieldType='unsigned', fieldPosition=4, isNullable=true}]},
	fields=[1000002, 26316, FIO, 52, 102]
}, TarantoolTupleImpl{
	spaceMetadata=TarantoolSpaceMetadata{spaceId=0, ownerId=0, spaceName='null', spaceFormatMetadata={profile_id=TarantoolFieldMetadataImpl{fieldName='profile_id', fieldType='unsigned', fieldPosition=0, isNullable=false}, bucket_id=TarantoolFieldMetadataImpl{fieldName='bucket_id', fieldType='unsigned', fieldPosition=1, isNullable=false}, fio=TarantoolFieldMetadataImpl{fieldName='fio', fieldType='string', fieldPosition=2, isNullable=false}, age=TarantoolFieldMetadataImpl{fieldName='age', fieldType='unsigned', fieldPosition=3, isNullable=false}, balance=TarantoolFieldMetadataImpl{fieldName='balance', fieldType='unsigned', fieldPosition=4, isNullable=true}}, spaceFormatMetadataAsList=[TarantoolFieldMetadataImpl{fieldName='profile_id', fieldType='unsigned', fieldPosition=0, isNullable=false}, TarantoolFieldMetadataImpl{fieldName='bucket_id', fieldType='unsigned', fieldPosition=1, isNullable=false}, TarantoolFieldMetadataImpl{fieldName='fio', fieldType='string', fieldPosition=2, isNullable=false}, TarantoolFieldMetadataImpl{fieldName='age', fieldType='unsigned', fieldPosition=3, isNullable=false}, TarantoolFieldMetadataImpl{fieldName='balance', fieldType='unsigned', fieldPosition=4, isNullable=true}]},
	fields=[1000003, 12041, FIO, 53, 103]
}, TarantoolTupleImpl{
	spaceMetadata=TarantoolSpaceMetadata{spaceId=0, ownerId=0, spaceName='null', spaceFormatMetadata={profile_id=TarantoolFieldMetadataImpl{fieldName='profile_id', fieldType='unsigned', fieldPosition=0, isNullable=false}, bucket_id=TarantoolFieldMetadataImpl{fieldName='bucket_id', fieldType='unsigned', fieldPosition=1, isNullable=false}, fio=TarantoolFieldMetadataImpl{fieldName='fio', fieldType='string', fieldPosition=2, isNullable=false}, age=TarantoolFieldMetadataImpl{fieldName='age', fieldType='unsigned', fieldPosition=3, isNullable=false}, balance=TarantoolFieldMetadataImpl{fieldName='balance', fieldType='unsigned', fieldPosition=4, isNullable=true}}, spaceFormatMetadataAsList=[TarantoolFieldMetadataImpl{fieldName='profile_id', fieldType='unsigned', fieldPosition=0, isNullable=false}, TarantoolFieldMetadataImpl{fieldName='bucket_id', fieldType='unsigned', fieldPosition=1, isNullable=false}, TarantoolFieldMetadataImpl{fieldName='fio', fieldType='string', fieldPosition=2, isNullable=false}, TarantoolFieldMetadataImpl{fieldName='age', fieldType='unsigned', fieldPosition=3, isNullable=false}, TarantoolFieldMetadataImpl{fieldName='balance', fieldType='unsigned', fieldPosition=4, isNullable=true}]},
	fields=[1000004, 4532, FIO, 54, 104]
}, TarantoolTupleImpl{
	spaceMetadata=TarantoolSpaceMetadata{spaceId=0, ownerId=0, spaceName='null', spaceFormatMetadata={profile_id=TarantoolFieldMetadataImpl{fieldName='profile_id', fieldType='unsigned', fieldPosition=0, isNullable=false}, bucket_id=TarantoolFieldMetadataImpl{fieldName='bucket_id', fieldType='unsigned', fieldPosition=1, isNullable=false}, fio=TarantoolFieldMetadataImpl{fieldName='fio', fieldType='string', fieldPosition=2, isNullable=false}, age=TarantoolFieldMetadataImpl{fieldName='age', fieldType='unsigned', fieldPosition=3, isNullable=false}, balance=TarantoolFieldMetadataImpl{fieldName='balance', fieldType='unsigned', fieldPosition=4, isNullable=true}}, spaceFormatMetadataAsList=[TarantoolFieldMetadataImpl{fieldName='profile_id', fieldType='unsigned', fieldPosition=0, isNullable=false}, TarantoolFieldMetadataImpl{fieldName='bucket_id', fieldType='unsigned', fieldPosition=1, isNullable=false}, TarantoolFieldMetadataImpl{fieldName='fio', fieldType='string', fieldPosition=2, isNullable=false}, TarantoolFieldMetadataImpl{fieldName='age', fieldType='unsigned', fieldPosition=3, isNullable=false}, TarantoolFieldMetadataImpl{fieldName='balance', fieldType='unsigned', fieldPosition=4, isNullable=true}]},
	fields=[1000005, 12689, FIO, 55, 105]
}, TarantoolTupleImpl{
	spaceMetadata=TarantoolSpaceMetadata{spaceId=0, ownerId=0, spaceName='null', spaceFormatMetadata={profile_id=TarantoolFieldMetadataImpl{fieldName='profile_id', fieldType='unsigned', fieldPosition=0, isNullable=false}, bucket_id=TarantoolFieldMetadataImpl{fieldName='bucket_id', fieldType='unsigned', fieldPosition=1, isNullable=false}, fio=TarantoolFieldMetadataImpl{fieldName='fio', fieldType='string', fieldPosition=2, isNullable=false}, age=TarantoolFieldMetadataImpl{fieldName='age', fieldType='unsigned', fieldPosition=3, isNullable=false}, balance=TarantoolFieldMetadataImpl{fieldName='balance', fieldType='unsigned', fieldPosition=4, isNullable=true}}, spaceFormatMetadataAsList=[TarantoolFieldMetadataImpl{fieldName='profile_id', fieldType='unsigned', fieldPosition=0, isNullable=false}, TarantoolFieldMetadataImpl{fieldName='bucket_id', fieldType='unsigned', fieldPosition=1, isNullable=false}, TarantoolFieldMetadataImpl{fieldName='fio', fieldType='string', fieldPosition=2, isNullable=false}, TarantoolFieldMetadataImpl{fieldName='age', fieldType='unsigned', fieldPosition=3, isNullable=false}, TarantoolFieldMetadataImpl{fieldName='balance', fieldType='unsigned', fieldPosition=4, isNullable=true}]},
	fields=[1000006, 4533, FIO, 56, 106]
}, TarantoolTupleImpl{
	spaceMetadata=TarantoolSpaceMetadata{spaceId=0, ownerId=0, spaceName='null', spaceFormatMetadata={profile_id=TarantoolFieldMetadataImpl{fieldName='profile_id', fieldType='unsigned', fieldPosition=0, isNullable=false}, bucket_id=TarantoolFieldMetadataImpl{fieldName='bucket_id', fieldType='unsigned', fieldPosition=1, isNullable=false}, fio=TarantoolFieldMetadataImpl{fieldName='fio', fieldType='string', fieldPosition=2, isNullable=false}, age=TarantoolFieldMetadataImpl{fieldName='age', fieldType='unsigned', fieldPosition=3, isNullable=false}, balance=TarantoolFieldMetadataImpl{fieldName='balance', fieldType='unsigned', fieldPosition=4, isNullable=true}}, spaceFormatMetadataAsList=[TarantoolFieldMetadataImpl{fieldName='profile_id', fieldType='unsigned', fieldPosition=0, isNullable=false}, TarantoolFieldMetadataImpl{fieldName='bucket_id', fieldType='unsigned', fieldPosition=1, isNullable=false}, TarantoolFieldMetadataImpl{fieldName='fio', fieldType='string', fieldPosition=2, isNullable=false}, TarantoolFieldMetadataImpl{fieldName='age', fieldType='unsigned', fieldPosition=3, isNullable=false}, TarantoolFieldMetadataImpl{fieldName='balance', fieldType='unsigned', fieldPosition=4, isNullable=true}]},
	fields=[1000007, 29736, FIO, 57, 107]
}, TarantoolTupleImpl{
	spaceMetadata=TarantoolSpaceMetadata{spaceId=0, ownerId=0, spaceName='null', spaceFormatMetadata={profile_id=TarantoolFieldMetadataImpl{fieldName='profile_id', fieldType='unsigned', fieldPosition=0, isNullable=false}, bucket_id=TarantoolFieldMetadataImpl{fieldName='bucket_id', fieldType='unsigned', fieldPosition=1, isNullable=false}, fio=TarantoolFieldMetadataImpl{fieldName='fio', fieldType='string', fieldPosition=2, isNullable=false}, age=TarantoolFieldMetadataImpl{fieldName='age', fieldType='unsigned', fieldPosition=3, isNullable=false}, balance=TarantoolFieldMetadataImpl{fieldName='balance', fieldType='unsigned', fieldPosition=4, isNullable=true}}, spaceFormatMetadataAsList=[TarantoolFieldMetadataImpl{fieldName='profile_id', fieldType='unsigned', fieldPosition=0, isNullable=false}, TarantoolFieldMetadataImpl{fieldName='bucket_id', fieldType='unsigned', fieldPosition=1, isNullable=false}, TarantoolFieldMetadataImpl{fieldName='fio', fieldType='string', fieldPosition=2, isNullable=false}, TarantoolFieldMetadataImpl{fieldName='age', fieldType='unsigned', fieldPosition=3, isNullable=false}, TarantoolFieldMetadataImpl{fieldName='balance', fieldType='unsigned', fieldPosition=4, isNullable=true}]},
	fields=[1000008, 17780, FIO, 58, 108]
}, TarantoolTupleImpl{
	spaceMetadata=TarantoolSpaceMetadata{spaceId=0, ownerId=0, spaceName='null', spaceFormatMetadata={profile_id=TarantoolFieldMetadataImpl{fieldName='profile_id', fieldType='unsigned', fieldPosition=0, isNullable=false}, bucket_id=TarantoolFieldMetadataImpl{fieldName='bucket_id', fieldType='unsigned', fieldPosition=1, isNullable=false}, fio=TarantoolFieldMetadataImpl{fieldName='fio', fieldType='string', fieldPosition=2, isNullable=false}, age=TarantoolFieldMetadataImpl{fieldName='age', fieldType='unsigned', fieldPosition=3, isNullable=false}, balance=TarantoolFieldMetadataImpl{fieldName='balance', fieldType='unsigned', fieldPosition=4, isNullable=true}}, spaceFormatMetadataAsList=[TarantoolFieldMetadataImpl{fieldName='profile_id', fieldType='unsigned', fieldPosition=0, isNullable=false}, TarantoolFieldMetadataImpl{fieldName='bucket_id', fieldType='unsigned', fieldPosition=1, isNullable=false}, TarantoolFieldMetadataImpl{fieldName='fio', fieldType='string', fieldPosition=2, isNullable=false}, TarantoolFieldMetadataImpl{fieldName='age', fieldType='unsigned', fieldPosition=3, isNullable=false}, TarantoolFieldMetadataImpl{fieldName='balance', fieldType='unsigned', fieldPosition=4, isNullable=true}]},
	fields=[1000009, 2785, FIO, 59, 109]
}, TarantoolTupleImpl{
	spaceMetadata=TarantoolSpaceMetadata{spaceId=0, ownerId=0, spaceName='null', spaceFormatMetadata={profile_id=TarantoolFieldMetadataImpl{fieldName='profile_id', fieldType='unsigned', fieldPosition=0, isNullable=false}, bucket_id=TarantoolFieldMetadataImpl{fieldName='bucket_id', fieldType='unsigned', fieldPosition=1, isNullable=false}, fio=TarantoolFieldMetadataImpl{fieldName='fio', fieldType='string', fieldPosition=2, isNullable=false}, age=TarantoolFieldMetadataImpl{fieldName='age', fieldType='unsigned', fieldPosition=3, isNullable=false}, balance=TarantoolFieldMetadataImpl{fieldName='balance', fieldType='unsigned', fieldPosition=4, isNullable=true}}, spaceFormatMetadataAsList=[TarantoolFieldMetadataImpl{fieldName='profile_id', fieldType='unsigned', fieldPosition=0, isNullable=false}, TarantoolFieldMetadataImpl{fieldName='bucket_id', fieldType='unsigned', fieldPosition=1, isNullable=false}, TarantoolFieldMetadataImpl{fieldName='fio', fieldType='string', fieldPosition=2, isNullable=false}, TarantoolFieldMetadataImpl{fieldName='age', fieldType='unsigned', fieldPosition=3, isNullable=false}, TarantoolFieldMetadataImpl{fieldName='balance', fieldType='unsigned', fieldPosition=4, isNullable=true}]},
	fields=[1000010, 25756, FIO, 60, 110]
}, TarantoolTupleImpl{
	spaceMetadata=TarantoolSpaceMetadata{spaceId=0, ownerId=0, spaceName='null', spaceFormatMetadata={profile_id=TarantoolFieldMetadataImpl{fieldName='profile_id', fieldType='unsigned', fieldPosition=0, isNullable=false}, bucket_id=TarantoolFieldMetadataImpl{fieldName='bucket_id', fieldType='unsigned', fieldPosition=1, isNullable=false}, fio=TarantoolFieldMetadataImpl{fieldName='fio', fieldType='string', fieldPosition=2, isNullable=false}, age=TarantoolFieldMetadataImpl{fieldName='age', fieldType='unsigned', fieldPosition=3, isNullable=false}, balance=TarantoolFieldMetadataImpl{fieldName='balance', fieldType='unsigned', fieldPosition=4, isNullable=true}}, spaceFormatMetadataAsList=[TarantoolFieldMetadataImpl{fieldName='profile_id', fieldType='unsigned', fieldPosition=0, isNullable=false}, TarantoolFieldMetadataImpl{fieldName='bucket_id', fieldType='unsigned', fieldPosition=1, isNullable=false}, TarantoolFieldMetadataImpl{fieldName='fio', fieldType='string', fieldPosition=2, isNullable=false}, TarantoolFieldMetadataImpl{fieldName='age', fieldType='unsigned', fieldPosition=3, isNullable=false}, TarantoolFieldMetadataImpl{fieldName='balance', fieldType='unsigned', fieldPosition=4, isNullable=true}]},
	fields=[1000011, 14569, FIO, 61, 111]
}, TarantoolTupleImpl{
	spaceMetadata=TarantoolSpaceMetadata{spaceId=0, ownerId=0, spaceName='null', spaceFormatMetadata={profile_id=TarantoolFieldMetadataImpl{fieldName='profile_id', fieldType='unsigned', fieldPosition=0, isNullable=false}, bucket_id=TarantoolFieldMetadataImpl{fieldName='bucket_id', fieldType='unsigned', fieldPosition=1, isNullable=false}, fio=TarantoolFieldMetadataImpl{fieldName='fio', fieldType='string', fieldPosition=2, isNullable=false}, age=TarantoolFieldMetadataImpl{fieldName='age', fieldType='unsigned', fieldPosition=3, isNullable=false}, balance=TarantoolFieldMetadataImpl{fieldName='balance', fieldType='unsigned', fieldPosition=4, isNullable=true}}, spaceFormatMetadataAsList=[TarantoolFieldMetadataImpl{fieldName='profile_id', fieldType='unsigned', fieldPosition=0, isNullable=false}, TarantoolFieldMetadataImpl{fieldName='bucket_id', fieldType='unsigned', fieldPosition=1, isNullable=false}, TarantoolFieldMetadataImpl{fieldName='fio', fieldType='string', fieldPosition=2, isNullable=false}, TarantoolFieldMetadataImpl{fieldName='age', fieldType='unsigned', fieldPosition=3, isNullable=false}, TarantoolFieldMetadataImpl{fieldName='balance', fieldType='unsigned', fieldPosition=4, isNullable=true}]},
	fields=[1000012, 877, FIO, 62, 112]
}, TarantoolTupleImpl{
	spaceMetadata=TarantoolSpaceMetadata{spaceId=0, ownerId=0, spaceName='null', spaceFormatMetadata={profile_id=TarantoolFieldMetadataImpl{fieldName='profile_id', fieldType='unsigned', fieldPosition=0, isNullable=false}, bucket_id=TarantoolFieldMetadataImpl{fieldName='bucket_id', fieldType='unsigned', fieldPosition=1, isNullable=false}, fio=TarantoolFieldMetadataImpl{fieldName='fio', fieldType='string', fieldPosition=2, isNullable=false}, age=TarantoolFieldMetadataImpl{fieldName='age', fieldType='unsigned', fieldPosition=3, isNullable=false}, balance=TarantoolFieldMetadataImpl{fieldName='balance', fieldType='unsigned', fieldPosition=4, isNullable=true}}, spaceFormatMetadataAsList=[TarantoolFieldMetadataImpl{fieldName='profile_id', fieldType='unsigned', fieldPosition=0, isNullable=false}, TarantoolFieldMetadataImpl{fieldName='bucket_id', fieldType='unsigned', fieldPosition=1, isNullable=false}, TarantoolFieldMetadataImpl{fieldName='fio', fieldType='string', fieldPosition=2, isNullable=false}, TarantoolFieldMetadataImpl{fieldName='age', fieldType='unsigned', fieldPosition=3, isNullable=false}, TarantoolFieldMetadataImpl{fieldName='balance', fieldType='unsigned', fieldPosition=4, isNullable=true}]},
	fields=[1000013, 2576, FIO, 63, 113]
}, TarantoolTupleImpl{
	spaceMetadata=TarantoolSpaceMetadata{spaceId=0, ownerId=0, spaceName='null', spaceFormatMetadata={profile_id=TarantoolFieldMetadataImpl{fieldName='profile_id', fieldType='unsigned', fieldPosition=0, isNullable=false}, bucket_id=TarantoolFieldMetadataImpl{fieldName='bucket_id', fieldType='unsigned', fieldPosition=1, isNullable=false}, fio=TarantoolFieldMetadataImpl{fieldName='fio', fieldType='string', fieldPosition=2, isNullable=false}, age=TarantoolFieldMetadataImpl{fieldName='age', fieldType='unsigned', fieldPosition=3, isNullable=false}, balance=TarantoolFieldMetadataImpl{fieldName='balance', fieldType='unsigned', fieldPosition=4, isNullable=true}}, spaceFormatMetadataAsList=[TarantoolFieldMetadataImpl{fieldName='profile_id', fieldType='unsigned', fieldPosition=0, isNullable=false}, TarantoolFieldMetadataImpl{fieldName='bucket_id', fieldType='unsigned', fieldPosition=1, isNullable=false}, TarantoolFieldMetadataImpl{fieldName='fio', fieldType='string', fieldPosition=2, isNullable=false}, TarantoolFieldMetadataImpl{fieldName='age', fieldType='unsigned', fieldPosition=3, isNullable=false}, TarantoolFieldMetadataImpl{fieldName='balance', fieldType='unsigned', fieldPosition=4, isNullable=true}]},
	fields=[1000014, 15141, FIO, 64, 114]
}, TarantoolTupleImpl{
	spaceMetadata=TarantoolSpaceMetadata{spaceId=0, ownerId=0, spaceName='null', spaceFormatMetadata={profile_id=TarantoolFieldMetadataImpl{fieldName='profile_id', fieldType='unsigned', fieldPosition=0, isNullable=false}, bucket_id=TarantoolFieldMetadataImpl{fieldName='bucket_id', fieldType='unsigned', fieldPosition=1, isNullable=false}, fio=TarantoolFieldMetadataImpl{fieldName='fio', fieldType='string', fieldPosition=2, isNullable=false}, age=TarantoolFieldMetadataImpl{fieldName='age', fieldType='unsigned', fieldPosition=3, isNullable=false}, balance=TarantoolFieldMetadataImpl{fieldName='balance', fieldType='unsigned', fieldPosition=4, isNullable=true}}, spaceFormatMetadataAsList=[TarantoolFieldMetadataImpl{fieldName='profile_id', fieldType='unsigned', fieldPosition=0, isNullable=false}, TarantoolFieldMetadataImpl{fieldName='bucket_id', fieldType='unsigned', fieldPosition=1, isNullable=false}, TarantoolFieldMetadataImpl{fieldName='fio', fieldType='string', fieldPosition=2, isNullable=false}, TarantoolFieldMetadataImpl{fieldName='age', fieldType='unsigned', fieldPosition=3, isNullable=false}, TarantoolFieldMetadataImpl{fieldName='balance', fieldType='unsigned', fieldPosition=4, isNullable=true}]},
	fields=[1000015, 11224, FIO, 65, 115]
}, TarantoolTupleImpl{
	spaceMetadata=TarantoolSpaceMetadata{spaceId=0, ownerId=0, spaceName='null', spaceFormatMetadata={profile_id=TarantoolFieldMetadataImpl{fieldName='profile_id', fieldType='unsigned', fieldPosition=0, isNullable=false}, bucket_id=TarantoolFieldMetadataImpl{fieldName='bucket_id', fieldType='unsigned', fieldPosition=1, isNullable=false}, fio=TarantoolFieldMetadataImpl{fieldName='fio', fieldType='string', fieldPosition=2, isNullable=false}, age=TarantoolFieldMetadataImpl{fieldName='age', fieldType='unsigned', fieldPosition=3, isNullable=false}, balance=TarantoolFieldMetadataImpl{fieldName='balance', fieldType='unsigned', fieldPosition=4, isNullable=true}}, spaceFormatMetadataAsList=[TarantoolFieldMetadataImpl{fieldName='profile_id', fieldType='unsigned', fieldPosition=0, isNullable=false}, TarantoolFieldMetadataImpl{fieldName='bucket_id', fieldType='unsigned', fieldPosition=1, isNullable=false}, TarantoolFieldMetadataImpl{fieldName='fio', fieldType='string', fieldPosition=2, isNullable=false}, TarantoolFieldMetadataImpl{fieldName='age', fieldType='unsigned', fieldPosition=3, isNullable=false}, TarantoolFieldMetadataImpl{fieldName='balance', fieldType='unsigned', fieldPosition=4, isNullable=true}]},
	fields=[1000016, 11284, FIO, 66, 116]
}, TarantoolTupleImpl{
	spaceMetadata=TarantoolSpaceMetadata{spaceId=0, ownerId=0, spaceName='null', spaceFormatMetadata={profile_id=TarantoolFieldMetadataImpl{fieldName='profile_id', fieldType='unsigned', fieldPosition=0, isNullable=false}, bucket_id=TarantoolFieldMetadataImpl{fieldName='bucket_id', fieldType='unsigned', fieldPosition=1, isNullable=false}, fio=TarantoolFieldMetadataImpl{fieldName='fio', fieldType='string', fieldPosition=2, isNullable=false}, age=TarantoolFieldMetadataImpl{fieldName='age', fieldType='unsigned', fieldPosition=3, isNullable=false}, balance=TarantoolFieldMetadataImpl{fieldName='balance', fieldType='unsigned', fieldPosition=4, isNullable=true}}, spaceFormatMetadataAsList=[TarantoolFieldMetadataImpl{fieldName='profile_id', fieldType='unsigned', fieldPosition=0, isNullable=false}, TarantoolFieldMetadataImpl{fieldName='bucket_id', fieldType='unsigned', fieldPosition=1, isNullable=false}, TarantoolFieldMetadataImpl{fieldName='fio', fieldType='string', fieldPosition=2, isNullable=false}, TarantoolFieldMetadataImpl{fieldName='age', fieldType='unsigned', fieldPosition=3, isNullable=false}, TarantoolFieldMetadataImpl{fieldName='balance', fieldType='unsigned', fieldPosition=4, isNullable=true}]},
	fields=[1000017, 20081, FIO, 67, 117]
}, TarantoolTupleImpl{
	spaceMetadata=TarantoolSpaceMetadata{spaceId=0, ownerId=0, spaceName='null', spaceFormatMetadata={profile_id=TarantoolFieldMetadataImpl{fieldName='profile_id', fieldType='unsigned', fieldPosition=0, isNullable=false}, bucket_id=TarantoolFieldMetadataImpl{fieldName='bucket_id', fieldType='unsigned', fieldPosition=1, isNullable=false}, fio=TarantoolFieldMetadataImpl{fieldName='fio', fieldType='string', fieldPosition=2, isNullable=false}, age=TarantoolFieldMetadataImpl{fieldName='age', fieldType='unsigned', fieldPosition=3, isNullable=false}, balance=TarantoolFieldMetadataImpl{fieldName='balance', fieldType='unsigned', fieldPosition=4, isNullable=true}}, spaceFormatMetadataAsList=[TarantoolFieldMetadataImpl{fieldName='profile_id', fieldType='unsigned', fieldPosition=0, isNullable=false}, TarantoolFieldMetadataImpl{fieldName='bucket_id', fieldType='unsigned', fieldPosition=1, isNullable=false}, TarantoolFieldMetadataImpl{fieldName='fio', fieldType='string', fieldPosition=2, isNullable=false}, TarantoolFieldMetadataImpl{fieldName='age', fieldType='unsigned', fieldPosition=3, isNullable=false}, TarantoolFieldMetadataImpl{fieldName='balance', fieldType='unsigned', fieldPosition=4, isNullable=true}]},
	fields=[1000018, 1605, FIO, 68, 118]
}, TarantoolTupleImpl{
	spaceMetadata=TarantoolSpaceMetadata{spaceId=0, ownerId=0, spaceName='null', spaceFormatMetadata={profile_id=TarantoolFieldMetadataImpl{fieldName='profile_id', fieldType='unsigned', fieldPosition=0, isNullable=false}, bucket_id=TarantoolFieldMetadataImpl{fieldName='bucket_id', fieldType='unsigned', fieldPosition=1, isNullable=false}, fio=TarantoolFieldMetadataImpl{fieldName='fio', fieldType='string', fieldPosition=2, isNullable=false}, age=TarantoolFieldMetadataImpl{fieldName='age', fieldType='unsigned', fieldPosition=3, isNullable=false}, balance=TarantoolFieldMetadataImpl{fieldName='balance', fieldType='unsigned', fieldPosition=4, isNullable=true}}, spaceFormatMetadataAsList=[TarantoolFieldMetadataImpl{fieldName='profile_id', fieldType='unsigned', fieldPosition=0, isNullable=false}, TarantoolFieldMetadataImpl{fieldName='bucket_id', fieldType='unsigned', fieldPosition=1, isNullable=false}, TarantoolFieldMetadataImpl{fieldName='fio', fieldType='string', fieldPosition=2, isNullable=false}, TarantoolFieldMetadataImpl{fieldName='age', fieldType='unsigned', fieldPosition=3, isNullable=false}, TarantoolFieldMetadataImpl{fieldName='balance', fieldType='unsigned', fieldPosition=4, isNullable=true}]},
	fields=[1000019, 7752, FIO, 69, 119]
}]}

```

</p>
</details> 

I haven't forgotten about:
- [ ] Tests
- [ ] Changelog
- [ ] Documentation
- [ ] Commit messages comply with the [guideline](https://www.tarantool.io/en/doc/latest/dev_guide/developer_guidelines/#how-to-write-a-commit-message)
- [ ] Cleanup the code for review. See [checklist](https://github.com/tarantool/cartridge-java/blob/master/docs/review-checklist.md)

Related issues:
Closes #406

### CODE REVIEW

**Review by iDneprov — APPROVED** [review]
LGTM

File: src/main/java/io/tarantool/driver/core/TarantoolResultImpl.java [hunk 1]
```diff
@@ -158,4 +158,11 @@ public ListIterator<T> listIterator(int index) {
     public List<T> subList(int fromIndex, int toIndex) {
         return this.tuples.subList(fromIndex, toIndex);
     }
+
+    @Override
+    public String toString() {
+        return "TarantoolResultImpl{" +
```
Comment (dkasimovskiy) [inline src/main/java/io/tarantool/driver/core/TarantoolResultImpl.java:164]: Давай единообразно будем использовать String.format

File: src/main/java/io/tarantool/driver/core/metadata/TarantoolFieldMetadataImpl.java [hunk 2]
```diff
@@ -65,4 +65,14 @@ public int getFieldPosition() {
     public boolean getIsNullable() {
         return isNullable;
     }
+
+    @Override
+    public String toString() {
+        return "TarantoolFieldMetadataImpl{" +
```
Comment (dkasimovskiy) [inline src/main/java/io/tarantool/driver/core/metadata/TarantoolFieldMetadataImpl.java:71]: Давай единообразно будем использовать String.format

**Review by dkasimovskiy — CHANGES_REQUESTED** [review]

File: src/main/java/io/tarantool/driver/core/TarantoolResultImpl.java (hunk 1 above)
Comment (ArtDu) [reply src/main/java/io/tarantool/driver/core/TarantoolResultImpl.java:164]: I replaced it with classic concat everywhere. We are building a perfomance software :)

https://stackoverflow.com/a/1281651

image

**Pull request merged**